File: model/Fornecedor.java
package model;

public class Fornecedor {
    private String cnpj;
    private String email;
    private String telefone;
    private String nome;
    
    public Fornecedor(String cnpj, String email, String telefone, String nome) {
        this.cnpj = cnpj;
        this.email = email;
        this.telefone = telefone;
        this.nome = nome;
    }
    
    public String getCnpj() {
        return cnpj;
    }
    public void setCnpj(String cnpj) {
        this.cnpj = cnpj;
    }
    public String getEmail() {
        return email;
    }
    public void setEmail(String email) {
        this.email = email;
    }
    public String getTelefone() {
        return telefone;
    }
    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }
    public String getNome() {
        return nome;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    @Override
    public String toString() {
        return "Fornecedor [cnpj=" + cnpj + ", email=" + email + ", telefone=" + telefone + ", nome=" + nome + "]";
    }
    
}

File: src/model/domain/Fornecedor.py
from sqlalchemy import Boolean, Column, Date, ForeignKey, Integer, String
from model.database.BaseORM import Base
from sqlalchemy.orm import relationship
from model.domain.Endereco import Endereco

class Fornecedor(Base):
    __tablename__ = 'fornecedor'

    id_fornecedor = Column("idfornecedor", Integer, primary_key=True, autoincrement=True)
    nome = Column("nome", String(45), nullable=False)
    cnpj = Column("cnpj", String(18), nullable=False)
    email = Column("email", String(45), nullable=False)
    telefone = Column("telefone", String(45), nullable=False)
    # Soft Delete
    data_delete = Column("datadelete", Date)
    foi_deletado = Column("foideletado", Boolean)
    id_endereco = Column("idendereco", Integer, ForeignKey('endereco.idendereco'), nullable=False)
    # Relacionamento com a tabela Endereco
    endereco = relationship("Endereco")